test_conformance/api/test_kernel_arg_info.cpp

```diff
@@ -542,6 +538,7 @@ size_t get_param_size(const std::string& arg_type, cl_device_id deviceID,
         cl_int err = clGetDeviceInfo(deviceID, CL_DEVICE_ADDRESS_BITS,
                                      sizeof(device_address_bits),
                                      &device_address_bits, NULL);
+        test_error(err, "clGetDeviceInfo");
```

Comment:
Just because we had a subtle bug in a previous CTS test: note that test_error() prints the error message then returns the error code as its return value, which probably isn't what is intended here.  I don't know what should be returned here necessarily if there is an OpenCL error, but consider test_error_ret() instead to return a specific value if there is an error.

Maybe:

```suggestion
        test_error_ret(err, "clGetDeviceInfo", 0);
```

Reply:
Good point, done.